src/faq.md

```diff
@@ -73,11 +73,10 @@ the VM and make deep changes to optimization of the language.
 
 ## Language
 
-{% comment %}
 ### Q. Isn't Dart a lot like Java?
 
-TODO: provide an up-to-date answer
-{% endcomment %}
+Dart has some similarities with Java. See the [Intro to Dart for Java
+Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart/index.html?index=..%2F..%2Findex#0) for an introduction to some of them.
```

Comment:
See the -> See the codelab

an introduction... -> (something more specific, maybe:)
examples of some of the differences between Dart and Java.